htdocs/class/cache/xoopscache.php

```diff
@@ -494,7 +494,7 @@ public function clear($check)
     /**
      * Cache Engine settings
      *
-     * @return array settings
+     * @return array|int settings
```

Comment:
I think the real confusion in the file is here:
```class XoopsCacheEngine
{
    /**
     * settings of current engine instance
     *
--> * @var int
     * @access public
     */
    public $settings;
```
That should be array, and never seems to be int